src/towncrier/newsfragments/663.bugfix.rst

```diff
@@ -0,0 +1 @@
+Markdown template: Render category headers as h2 when no sections are present.
```

Comment:
Just a small comment... and mauybe this should be a `.removal` type of fragment.

It should explain the existing users should manually update existing NEWS file to match the new H2 format.


```suggestion
When no sections are present, the default Markdown template now renders the category headers as H2. In previous versions it was rendered as H3.
```